
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Bad request
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.626+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "GET_GetAccounts_400", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class GETGetAccounts400 extends Exception {

    private com.laegler.openbanking.soap.model.OBErrorResponse1 getGetAccounts400;

    public GETGetAccounts400() {
        super();
    }

    public GETGetAccounts400(String message) {
        super(message);
    }

    public GETGetAccounts400(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public GETGetAccounts400(String message, com.laegler.openbanking.soap.model.OBErrorResponse1 getGetAccounts400) {
        super(message);
        this.getGetAccounts400 = getGetAccounts400;
    }

    public GETGetAccounts400(String message, com.laegler.openbanking.soap.model.OBErrorResponse1 getGetAccounts400, java.lang.Throwable cause) {
        super(message, cause);
        this.getGetAccounts400 = getGetAccounts400;
    }

    public com.laegler.openbanking.soap.model.OBErrorResponse1 getFaultInfo() {
        return this.getGetAccounts400;
    }
}
